data/governments.txt

```diff
@@ -462,6 +517,8 @@ government "Syndicate"
 	
 	"player reputation" 2
 	"attitude toward"
+		"Bounty Hunter Escort" .3
+		`Legitimate Businessmen` .3
```

Comment:
Throughout this file: <code>\`Legitimate Businessmen\`</code> (and some other new govements) have been used  with grave accents (\`), while all other governments are listed with double quotes (")

Reply:
That does not matter.

Reply:
Show me in the coding style guide where this is stated as a requirement, and I will fix it.

Reply:
Fixed by Amazinite's request.